BotRoyaleUnity/Assets/Scripts/ShrinkArena.cs

```diff
@@ -86,16 +85,6 @@ private IEnumerator ShrinkSequence()
             yield return null;
         }
 
-        /*
-        while (elapsedWallCollapseTime < WallCollapseLength)
```

Comment:
There are some compiler warnings about unused variables, I think because this code is commented out.
Look into fixing these warnings, I think it should be fine to remove this code since we're not using it